client_example/11_grouped_conv_bwd_weight/grouped_conv2d_bwd_weight_fp16.cpp

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#include "common.hpp"
+
+#include "ck/ck.hpp"
+#include "ck/tensor_operation/gpu/device/tensor_layout.hpp"
+
+using InDataType  = ck::half_t;
+using WeiDataType = ck::half_t;
+using OutDataType = ck::half_t;
+
+using InLayout  = ck::tensor_layout::convolution::GNHWC;
```

Comment:
These tensor layout are not standard layout for grouped conv.

The standard layout are these packed layout:
NHWGC, GKYXC (or maybe KYXGC, I'm not entirely sure), NHWGK

However, we express these stand layout as non-packed layout
G_NHW_C, G_KYX_C, and G_NHW_K. The ```_``` means the dimensions before and after ```_``` are not packed, so you specify a stride.

We can discuss in the meeting if you bring up this issue

https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/274108d6e6345f6a4ef4e5d9b23dbfd3a8c2b2f8/example/30_grouped_conv_fwd_multiple_d/run_grouped_conv_fwd_bias_relu_add_example.inc#L14-L27

Reply:
@aosewski Has Chao's comment been resolved?